executor/common.h

```diff
@@ -17,6 +17,13 @@
 
 #if GOOS_freebsd || GOOS_test && HOSTGOOS_freebsd
 #include <sys/endian.h> // for htobe*.
+#elif GOOS_darwin
+#include <libkern/OSByteOrder.h>
```

Comment:
[Looks like](https://stackoverflow.com/questions/27073837/fatal-error-endian-h-file-not-found) we can do better with:
```c
#include <machine/endian.h>
```

Reply:
That header does exist, but does not contain these function, so no, I don't think we can do better here.